en/tween/tween-interface.md

```diff
@@ -0,0 +1,80 @@
+# Tween Interface
+
+## Properties and Interface
+
+### Interface
+
+| Interface         | Description                                     |
+| :---------------- | :------------------------------------------ |
+| **tag**           | Add a tag of type `number` to the current tween |
+| **to**            | Adds an interval action that performs **absolute value** calculations on properties |
+| **by**            | Adds an interval action that performs **relative value** calculations on properties |
+| **set**           | Add a transient action that **directly sets the target property** |
+| **delay**         | Add a **delay time** to a transient action |
+| **call**          | Add a **callback** transient
+| **target**        | Add a **directly set tween target** transient
+| **union**         | **package the contextual tween action into one** |
+| **then**          | **insert a tween into the tween queue** |
+| **repeat**        | **execute several times** (previously repeated several times, please adapt in time) |
```

Comment:
Define the number of times for the motion to be executed (In previous versions, this is used to define the number of times the motion is repeated.)